warehouse/migrations/versions/b265ed9eeb8a_fully_deprecate_legacy_distribution_.py

```diff
@@ -0,0 +1,42 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Fully deprecate legacy distribution types
+
+Revision ID: b265ed9eeb8a
+Revises: 5c029d9ef925
+Create Date: 2020-03-12 17:51:08.447903
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+revision = "b265ed9eeb8a"
+down_revision = "5c029d9ef925"
+
+
+def upgrade():
+    op.drop_column("projects", "allow_legacy_files")
+
+
+def downgrade():
```

Comment:
This should probably just fail, going backwards doesn't make sense here because re-adding this isn't going to restore the values in the database that marked whether a project could accept legacy uploads or not.